catalog.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Нёрдс</title>
+  </head>
+  <body>
+      <header>
+        <nav>
+          <a href="#"></a>
+            <strong>Nërd's</strong>
+        </a>
+        <ul>
+          <li><a href="#">Студия</a></li>
+          <li><a href="#">Клиенты</a></li>
+          <li><a href="#">Магазин</a></li>
+          <li><a href="#">Контакты</a></li>
+          <a href="#"><img src="https://via.placeholder.com/15x11" width="15" height="11" alt="иконка">Корзина</a>
+        </ul>
+      </nav>
+      </header>
+      <main>
+          <h2>Фильтр товаров</h2>
+          <span><strong>СТОИМОСТЬ:</strong></span>
+          <form method="get" action="https://echo.htmlacademy.ru">
+              <fieldset>
+                  <ul>
+                      <li>
+                          <input type="range" id="price" name="price" min="0" max="15000" />
```

Comment:
Инпут это одиночной тег, его можно (даже было бы лучше) не закрывать
```suggestion
                          <input type="range" id="price" name="price" min="0" max="15000">
```